### PR TITLE
Added typesystem recipe

### PR DESCRIPTION
**Pull request by k-dominik**
Hey @conda-forge/help-python, this is a PR for [typesystem](https://github.com/encode/typesystem) another package developed by the encode team. This package is also a dependency for [apistar](https://github.com/encode/apistar), ([feedstock](https://github.com/conda-forge/apistar-feedstock)).

### CODE REVIEW

File: recipes/typesystem/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,1101 @@
+
```
Comment (marcelotrevisani) [inline recipes/typesystem/LICENSE.md]: I think this license file is incorrect, right!?

**Comment by marcelotrevisani** [comment]
You have a python error here
```
import typesystem/tokenize
```
it is in your test

File: recipes/typesystem/LICENSE.md (hunk 1 above)
Comment (marcelotrevisani) [inline recipes/typesystem/LICENSE.md]: I think this license file is incorrect, right!?

File: recipes/typesystem/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - typesystem
+    - typesystem/tokenize
```
Comment (marcelotrevisani) [inline recipes/typesystem/meta.yaml]: This should be
```
- typesystem.tokenize
```

File: recipes/typesystem/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+about:
+  home: https://github.com/encode/typesystem
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.md
```
Comment (marcelotrevisani) [inline recipes/typesystem/meta.yaml]: You can raise an issue to this project to also add the license file in the pypi

File: recipes/typesystem/LICENSE.md (hunk 1 above)
Comment (k-dominik) [reply recipes/typesystem/LICENSE.md]: yes, totally, thank you! :)

**Comment by k-dominik** [comment]
Dear @marcelotrevisani, thank you so much for taking the time to look over the PR and catching my mistakes there. I think it should be fine now, but let's see what the CI has to say :).

Concerning the license file in pypi: will do, for the current version, however, the file is needed.

Thank you very much again!

**Comment by marcelotrevisani** [comment]
Thanks! @k-dominik

**Pull request merged**